.circleci/config.yml

```
@@ -191,7 +191,7 @@ jobs:
   unit_tests_node8:
     executor:
       name: node
-      image: "8"
+      image: "8.9.0"
```

```suggestion
      image: "8.0.0"
```

Shouldn't be the first LTS release but the first release of Node 8 itself since we document support for Node 8.0.0 as a minimum

---

Source of image tag was https://circleci.com/docs/2.0/docker-image-tags.json

---

@sidharthachatterjee Like @m-allanson already said: `8.9.0` was the first LTS version [according  to Node's blog](https://nodejs.org/fr/blog/release/v8.9.0)

---

Yup, that’s true! Unfortunately when we decided to deprecate Node 6 support, we promised that the lowest Node version we’d support was Node 8.0.0.

---

Right now that's not true anymore tho, since `8.0.0` and even `8.9.0` is failing tests

---

Yup, considering we’re just a few days away from EOL for Node 8, I guess let’s put this PR on hold since we’ll likely deprecate 8 for 10 LTS then.